tools/testsys/src/aws_resources.rs

```diff
@@ -73,21 +140,21 @@ impl AwsK8s {
     }
 
     /// Bottlerocket cluster naming convention.
-    fn cluster_name(&self, suffix: &str) -> String {
+    fn cluster_name(&self) -> String {
```

Comment:
It was not needed anymore migration testing. It was left in case an issue in testsys (running upgrade downgrade and conformance on the same cluster) was not solved, but a solution for the issue is currently in review.

Reply:
I can make a separate pr/commit to remove that if needed.

Reply:
I'm fine with the update, no need for a new PR (folks may have a different point of view), but it was strange to me to update a method that was just added. I'll let you mark this conversation as resolved so that you don't miss this message.